src/main/java/com/device/shop/controller/CartItemController.java

```diff
@@ -0,0 +1,48 @@
+package com.device.shop.controller;
+
+import com.device.shop.entity.CartItem;
+import com.device.shop.entity.Product;
+import com.device.shop.service.CartItemService;
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+
+import java.util.List;
+
+@Controller
+@AllArgsConstructor
+public class CartItemController {
+
+    CartItemService cartItemService;
+
+    @PostMapping("/cart/{id}/items")
```

Comment:
What is the difference between items and products in the urls of this Controller?